Test Selenium website

HaykMelikyan opened the pull request:
Set up selenium dependency. Check the Selenium Grid download version and test search in selenium.dev website.

HaykMelikyan committed:
Add Selenium dependency in pom.xml

HaykMelikyan committed:
Add chromedriver to directory

HaykMelikyan committed:
Test Selenium Grid download version and search in Selenium website

chandranik90 commented on src/test/java/SeleniumWebsiteTest.java:
```diff
@@ -0,0 +1,39 @@
+        browser.findElement(By.cssSelector("a.nav-item[href = '/downloads']")).click();
+        String latestVersion = browser.findElement(By.xpath(
+                "//p[contains(normalize-space(),'Latest stable version')]/a")).getText();
+        Assert.assertEquals(latestVersion, "3.141.59", "Latest version is 3.141.59.");
```
Assertion message should contain a negative statement because it's an error message

chandranik90 commented on src/test/java/SeleniumWebsiteTest.java:
```diff
@@ -0,0 +1,39 @@
+
+        browser.get("https://www.selenium.dev/");
+        browser.findElement(By.cssSelector("a.nav-item[href = '/downloads']")).click();
+        String latestVersion = browser.findElement(By.xpath(
```
It's better to put `Thread.sleep`-s between actions, when the page is being reloaded or rendering started

chandranik90 replied:
It's a good practice to use `actual` and `expected` prefixes in variables' names. In this case, the `latestVersion` is your actual value. So, it's better to rename it to `actualLatestVersion`

HaykMelikyan committed:
Make corrections

- Change assertion messages to negative statements
- Add Thread.sleep between actions
- Change variable's name, which represent actual result
